### DISCUSSION TITLE
Little guide

### DISCUSSION THREAD

**Discussion by Oliwer5002** [Chia-Network/chia-blockchain#14402] [Support]
Hello everybody , i'm new.  I wan't to start chia mining and i have few questions. 
1. Witch of types it's better SSD or HDD
2. What's better have 100x HDD 2 tb's discs or 200x HDD 1tb's 
3. Does disk read and write speed matter?
4.What i have to know while starting ?

**Comment by shaneo257** [comment]
1. When plotting, its better to have a SSD at a temp drive
2. Its better to use the largest capacity HDD's you can afford
3. No
4. Have a read through https://docs.chia.net/quick-start-guide, and it should answer most of your questions

**Reply by Oliwer5002** [reply]
So, I can connect the disks using usb to sata adapter ?
